app/src/main/res/layout/nav_list.xml

```diff
@@ -69,6 +69,5 @@
         android:dividerHeight="0dp"
         android:paddingBottom="@dimen/list_vertical_padding"
         android:paddingTop="8dp"
-        android:scrollbarStyle="outsideOverlay"
-        tools:listitem="@layout/nav_listitem" />
```

Comment:
I think the `tools:listitem` attribute is just for displaying in Android Studio. It should not break anything.